Algorithm.md

```diff
@@ -0,0 +1,193 @@
+# Drawing Shapes with Computer Graphics Algorithms:
+
+> This file contains the codes for drawing various shapes using computer graphics algorithms:
+
+
+## 1. Digital Differential Analyzer (DDA) Line Code:
+```c
+// This C++ program uses OpenGL to draw a line based on the DDA (Digital Differential Analyzer) Line Drawing Algorithm, 
+// which calculates intermediate points between two user-defined endpoints to create a smooth line segment.
+
+
+
+#include <iostream>
+#include <GL/glut.h>
+#include <GL/glu.h>
+#include <cmath>
+
+int x1, Y1, x2, y2;
```

Comment:
y1 la Y capital zalay to small kr.

Reply:
Its done purposefully to avoid conflict between built-in y1 keyword and the one variable used in the code.